pom.xml

```diff
@@ -5,9 +5,9 @@
   <parent>
     <groupId>com.hotels</groupId>
     <artifactId>hotels-oss-parent</artifactId>
-    <version>2.0.6</version>
+    <version>2.3.3</version>
```

Comment:
Needs a changelog line describing this change

Reply:
Should I say what version it was changed from?

Reply:
Might as well - "Upgraded hotels-oss-parent to  2.3.3 (was 2.0.6)" or something similar.